includes/discovery/sensors/frequency/linux.inc.php

```diff
@@ -3,24 +3,22 @@
  * raspberry pi frequencies
  * requires snmp extend agent script from librenms-agent
  */
-$raspberry = snmp_get($device, 'HOST-RESOURCES-MIB::hrSystemInitialLoadParameters.0', '-Osqnv');
+$sensor_type = "raspberry_freq";
+$oid = '.1.3.6.1.4.1.8072.1.3.2.4.1.2.9.114.97.115.112.98.101.114.114.121.';
```

Comment:
Why wasn't this walked instead of doing for with several gets?

What if .6 is not a valid data point, but .7 is?